fe4u-boilerplate-main/src/js/API/teacherApi.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+const API = require("./API");
+
+const API_RANDOM_USER_URL = 'https://randomuser.me/api/';
+const RandomUsers = new API(API_RANDOM_USER_URL);
+
+class teacherApi {
+
+    constructor({seed}) {
+        this.seed = seed;
+    }
+
+    getPath({options}) {
+        return API_RANDOM_USER_URL + API.toStringOptions(options);
+    }
+
+    async getStartPage({results}) {
+        const options = {
+            page: 1,
+            seed: this.seed,
+            results: results,
+        };
+        console.log(this.getPath({options}));
```

Comment:
Usually we leave `console`s in `try...catch...` blocks only. Sometimes in tests.
All other `console`s are useless and should be deleted before PR creation.